homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.                                    INNER JOIN person ON person.id=roles.personId''', con)
</pre><p>Does Inner Join here is correct?, What if some movie does not have actor record in our table?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='138'/>